index.html

```diff
@@ -23,22 +23,142 @@
             </ul>
           </div>
         </nav>
-    </header>
+      </header>
     <section class="main">
-        <div class="content">
-            <h1>I'm Marie Grace.<br>Glad to see <br> you!</h1>
-            <p class="presentation">I'm a software developer and I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
-            <p class="connect">LET'S CONNECT</p>
-         <div class="icons">
-                <ul>
-                    <li><a href=""><i class="fa fa-twitter"></i></a></li>
-                    <li><a href=""><i class="fa fa-linkedin"></i></a></li>
-                    <li><a href=""><i class="fa fa-medium"></i></a></li>
-                    <li><a href=""><i class="fa fa-github"></i></a></li>
-                    <li><a href=""><i class="fa fa-angellist"></i></a></li>
-                </ul>
-            </div>
+      <div class="content">
+          <h1>I'm Marie Grace.<br>Glad to see <br> you!</h1>
+          <p class="presentation">I'm a software developer and I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+          <p class="connect">LET'S CONNECT</p>
+       <div class="icons">
+              <ul>
+                  <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                  <li><a href=""><i class="fa fa-linkedin"></i></a></li>
+                  <li><a href=""><i class="fa fa-medium"></i></a></li>
+                  <li><a href=""><i class="fa fa-github"></i></a></li>
+                  <li><a href=""><i class="fa fa-angellist"></i></a></li>
+              </ul>
+          </div>
+      </div>
+  </section>
+  <section class="works">
+      <img class="photo" src="images/tonic project.png">
+      <h1 class="title">Tonic</h1>
+      <div>
+        <ul class="canopy">
+          <li class="can"><b>CANOPY</b></li>
+          <li class="dev"><b>Back End Dev</b></li>
+          <li class="dev"> <b> 2015</b></li>
+        </ul>
+      </div>
+      <p class="presentation">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="languages">
+        <ul>
+          <li class="lang">HTML</li>
+          <li class="lang">CSS</li>
+          <li class="lang">JavaScript</li>
+        </ul>
+      </div>
+      <p class="button">See project</p>
```

Comment:
Please as good practice is important that you make the "See Project" wrapped in a `button` or an `a` tag, not a paragraph within a `p` tag. This is important for accessibility and user experience. please consider to update these commented line accordingly.